src/react-collapsed.js

```diff
@@ -86,6 +86,8 @@ export default function useCollapse(initialConfig = {}) {
       // it's safe to clear all style overrides
       if (height === styles.height) {
         setStyles({});
+      } else {
+        setStyles({height});
```

Comment:
We should combine this height property with the the current styles, otherwise we lose the transition timing function and all that. 

```suggestion
        setStyles(oldStyles => ({...oldStyles, height}));
```